src/encoded/static/components/experiment.js

```diff
@@ -769,8 +806,19 @@ const ExperimentComponent = ({ context, auditIndicators, auditDetail }, reactCon
                 <ReplicateTable condensedReplicates={condensedReplicates} replicationType={context.replication_type} />
             : null}
 
-            {/* Display the file widget with the facet, graph, and tables */}
-            <FileGallery context={context} encodevers={encodevers} anisogenic={anisogenic} />
+            {/* Display the file widget with the facet, graph, and tables for Experiment and FunctionalCharacterizationExperiment only. */}
+            {!isEnhancerExperiment ?
+                <FileGallery context={context} encodevers={encodevers} anisogenic={anisogenic} />
+            : null}
+
+            {/* Display the biosample table for TransgenicEnhancerExperiment only. */}
+            {isEnhancerExperiment ?
+                <BiosampleTable
+                    title="Biosamples"
+                    items={context.biosamples}
+                    total={context.biosamples.length}
+                />
+            : null}
```

Comment:
`context.biosample_ontology` gets embedded, but context.biosamples.biosample_ontology doesn’t, so you’re missing the data for couple columns in the biosample table. Unless you can embed biosamples.biosample_ontology, you’ll have to do a request to the server to display this table.

Reply:
do you mean like this? 
                `<FetchedItems
                    title="Biosamples"
                    url={`/search/?type=Biosample&DATASET???`}
                    Component={BiosampleTable}
                />`
The only problem is that in the url I don't think biosample has any `dataset` properties. Unless I am mistaken.

Reply:
Something like that, yes. I think you’ll need to put together a URL that just lists all the biosample URLs in the experiment, like:

http://localhost:6543/search/?type=Biosample&@id=/biosamples/ENCBS356AAA/&@id=/biosamples/ENCBS888RUL/

…and just keep adding on for each biosample in the experiment. There’s a limit to what you can put into the query string though, so you’ll need to do something different if the list of biosamples could get big, like over a 100 or so.